packages/@okta/vuepress-site/docs/guides/oin-oidc-guide/use-cases/index.md

```diff
@@ -0,0 +1,3 @@
+---
+title: Use-cases
```

Comment:
What is this?

Reply:
A place holder for future content - the use cases didn't get filled out by anyone, so Unni said to shelve it for the initial release. I'll remove it for now.